pkg/common/cns-lib/volume/listview.go

```diff
@@ -181,8 +187,10 @@ func (l *ListViewImpl) RemoveTask(ctx context.Context, taskMoRef types.ManagedOb
 	return nil
 }
 
-func (l *ListViewImpl) isClientValid() error {
+func (l *ListViewImpl) isClientValid(reconnect bool) error {
```

Comment:
note: I've set the `reconnect` param to false for calls from `AddTask()` and `RemoveTask()` as I want the re-connection to happen via the call from `listenToTaskUpdates()` method as it will also re-create the `ListView` object which is what we want to do.

Reply:
can you add this as comment for func isClientValid